src/gui/inputdevices/MouseInputHandler.cpp

```diff
@@ -54,9 +54,16 @@ bool MouseInputHandler::handleImpl(InputEvent* event)
 	 * Trigger motion actions
 	 */
 	// Trigger motion action when pen/mouse is pressed and moved
-	if (this->deviceClassPressed && event->type == MOTION_EVENT) //mouse or pen moved
+	if (event->type == MOTION_EVENT) //mouse or pen moved
 	{
-		this->actionMotion(event);
+		if (this->deviceClassPressed)
+		{
```

Comment:
I tried :
```
XournalppCursor* cursor = xournal->view->getCursor();
			cursor->updateCursor();
```
here to get teh custom cursor during mouse down. What do you think?

Reply:
I did not really get what you mean. Could you elaborate?